corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_view.js

```diff
@@ -134,7 +134,11 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
         initialize: function (options) {
             this.tiles = options.tiles;
             this.styles = options.styles;
-            generateCaseTileStyles(options.tiles);
+            if (!_.isNull(this.options.tiles)) {
+                $.getScript("../../../../static/css-grid-polyfill-binaries/css-polyfills.js", function() {
```

Comment:
mmm, i don't think this'll work in production. i think we should use the django `static` method, though that only works in templates. there are a couple of options, we can define an hq module with `hqDefine` and set some common formplayer data like the url to the css polyfill. if it can wait we can discuss on tuesday